README.md

```diff
@@ -32,6 +32,7 @@ plugins:
         with:
           # Define configurable items here and the kernel will pass these to the plugin.
           configurableResponse: "Hello, is it me you are looking for?"
+          customStringsUrl: "https://raw.githubusercontent.com/dev/new-pugin/development/strings.json"
```

Comment:
Why don't we pass a real `json` file here from the https://github.com/ubiquibot/plugin-template repo?

Reply:
@rndquu You're right. so? https://raw.githubusercontent.com/ubiquibot/plugin-template/development/strings.json